modules/using-lab-tuning.adoc

```diff
@@ -0,0 +1,156 @@
+.Verification
+The details page for the registered base model opens.
+
+== Starting a LAB-tuning run from the registered model
+
+After registering a base model, you can start a LAB-tuning run to create your customized large language model (LLM). When you configure the run, you will provide the following details:
+
+* The Git repository URL for your taxonomy.
+* The URLs where the teacher and judge models are deployed and the credentials needed to access them.
+* The hardware profile to use for data generation, training, and evaluation.
+* Any hyperparameters to apply to the tuning process.
+* The location to store the LAB-tuned model and the credentials needed to access the storage location.
+
+.Prerequisites
+* You have logged in to {productname-short}.
+* Your cluster administrator has configured LAB-tuning in your cluster, as described in _Enabling LAB-tuning_. 
+* You have enabled the LAB-tuning features in your current session, as described in _Making LAB-tuning features visible_.
+* You have access to an available model registry in your deployment.
+* You have created your taxonomy, prepared an OCI storage location for the LAB-tuned model, created a data science project, and deployed teacher and judge models, as described in _Preparing LAB-tuning resources_.
+* You have registered a base model with the `LAB starter` label from the model catalog, as described in _Registering a base model_.
+
+.Procedure
+. From the {productname-short} dashboard, click *Models* -> *Model registry*.
++
+The *Model registry* page opens.
+. Click the name of the model you registered.
++
+The details page for the registered model opens.
+. On the *Versions* tab of the model details page, click the name of the version you want to LAB-tune.
++
+The details page for the version opens.
+. Review the version details.
+. Click *LAB-tune*.
++
+The *Start a LAB-tuning run* dialog opens.
+. Select your project from the *Data science project* drop-down list.
+. Click *Continue to run details*.
++
+. The *Start a LAB-tuning run* page opens.
+. Review the *Project details*, *Pipeline details*, and *Base model* sections for accuracy.
+. In the *Taxonomy details* section, enter your *Taxonomy GIT URL*. 
+. If your Git repository requires authentication, select either the *SSH key* or *Username and token* method and enter the appropriate credentials.
+. In the *LAB teacher model* section, if your teacher model requires token authentication, select *Authenticated endpoint* and enter the *Endpoint*, *Token*, and *Model name*. If your teacher model does not require token authentication, select *Unauthenticated endpoint* and enter the *Endpoint* and *Model name*. 
++
+[TIP]
+====
+To find token authentication for your judge and teacher models, go to the *Models* tab of your data science project. For *Endpoint* and *Model name*, click *Internal and external endpoint details* for your model. For *Token*, expand the section for your model and find the *Token authentication* section.
```

Comment:
Important: for both models, users need to add `/v1` at the end of the url of the endpoint provided by RHOAI when configuring the LAB-tuning process

Example:  https://mixtral-my-project.apps.my-cluster.com/v1